lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
@marcparadise This API is throwing an error for me when it applies middleware to original response. The user does however get added to organization as a member though. Unfortunately only the first member in list gets added because recipe crashes. 

```
[2015-02-25T06:33:59+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
root@poliva2-chef12-standalone-thor:~# cat /var/chef/cache/chef-stacktrace.out
Generated at 2015-02-25 06:33:58 +0000
Chef::Exceptions::JSON::ParseError: chef_organization[blackberry] (bb_chef12::_manage_rbac line 74) had an error: Chef::Exceptions::JSON::ParseError: Top level JSON object must be a Hash or Array. (actual: NilClass)
/opt/chefdk/embedded/apps/chef/lib/chef/json_compat.rb:62:in `from_json'
/opt/chefdk/embedded/apps/chef/lib/chef/http/json_output.rb:53:in `handle_response'
/opt/chefdk/embedded/apps/chef/lib/chef/http.rb:232:in `block in apply_response_middleware'
/opt/chefdk/embedded/apps/chef/lib/chef/http.rb:230:in `each'
/opt/chefdk/embedded/apps/chef/lib/chef/http.rb:230:in `inject'
/opt/chefdk/embedded/apps/chef/lib/chef/http.rb:230:in `apply_response_middleware'
/opt/chefdk/embedded/apps/chef/lib/chef/http.rb:144:in `request'
/opt/chefdk/embedded/apps/chef/lib/chef/http.rb:126:in `post'
/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/cheffish-0.9.2/lib/chef/provider/chef_organization.rb:57:in `block (3 levels) in <class:ChefOrganization>'
/opt/chefdk/embedded/apps/chef/lib/chef/mixin/why_run.rb:52:in `call'
/opt/chefdk/embedded/apps/chef/lib/chef/mixin/why_run.rb:52:in `add_action'
/opt/chefdk/embedded/apps/chef/lib/chef/provider.rb:156:in `converge_by'
/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/cheffish-0.9.2/lib/chef/provider/chef_organization.rb:55:in `block (2 levels) in <class:ChefOrganization>'
/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/cheffish-0.9.2/lib/chef/provider/chef_organization.rb:53:in `each'
/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/cheffish-0.9.2/lib/chef/provider/chef_organization.rb:53:in `block in <class:ChefOrganization>'
/opt/chefdk/embedded/apps/chef/lib/chef/provider/lwrp_base.rb:138:in `instance_eval'
/opt/chefdk/embedded/apps/chef/lib/chef/provider/lwrp_base.rb:138:in `block in action'
/opt/chefdk/embedded/apps/chef/lib/chef/provider.rb:121:in `run_action'
/opt/chefdk/embedded/apps/chef/lib/chef/resource.rb:648:in `run_action'
/opt/chefdk/embedded/apps/chef/lib/chef/runner.rb:49:in `run_action'
/opt/chefdk/embedded/apps/chef/lib/chef/runner.rb:81:in `block (2 levels) in converge'
/opt/chefdk/embedded/apps/chef/lib/chef/runner.rb:81:in `each'
/opt/chefdk/embedded/apps/chef/lib/chef/runner.rb:81:in `block in converge'
/opt/chefdk/embedded/apps/chef/lib/chef/resource_collection.rb:98:in `block in execute_each_resource'
/opt/chefdk/embedded/apps/chef/lib/chef/resource_collection/stepable_iterator.rb:116:in `call'
/opt/chefdk/embedded/apps/chef/lib/chef/resource_collection/stepable_iterator.rb:116:in `call_iterator_block'
/opt/chefdk/embedded/apps/chef/lib/chef/resource_collection/stepable_iterator.rb:85:in `step'
/opt/chefdk/embedded/apps/chef/lib/chef/resource_collection/stepable_iterator.rb:104:in `iterate'
/opt/chefdk/embedded/apps/chef/lib/chef/resource_collection/stepable_iterator.rb:55:in `each_with_index'
/opt/chefdk/embedded/apps/chef/lib/chef/resource_collection.rb:96:in `execute_each_resource'
/opt/chefdk/embedded/apps/chef/lib/chef/runner.rb:80:in `converge'
/opt/chefdk/embedded/apps/chef/lib/chef/client.rb:345:in `converge'
/opt/chefdk/embedded/apps/chef/lib/chef/client.rb:431:in `do_run'
/opt/chefdk/embedded/apps/chef/lib/chef/client.rb:213:in `block in run'
/opt/chefdk/embedded/apps/chef/lib/chef/client.rb:207:in `fork'
/opt/chefdk/embedded/apps/chef/lib/chef/client.rb:207:in `run'
/opt/chefdk/embedded/apps/chef/lib/chef/application.rb:236:in `run_chef_client'
/opt/chefdk/embedded/apps/chef/lib/chef/application/solo.rb:226:in `block in run_application'
/opt/chefdk/embedded/apps/chef/lib/chef/application/solo.rb:218:in `loop'
/opt/chefdk/embedded/apps/chef/lib/chef/application/solo.rb:218:in `run_application'
/opt/chefdk/embedded/apps/chef/lib/chef/application.rb:55:in `run'
/opt/chefdk/embedded/apps/chef/bin/chef-solo:25:in `<top (required)>'
/usr/bin/chef-solo:33:in `load'
``` 

Is the api fixed in different chef version? Using chefdk 0.3.5 which uses `11.18.0.rc.1`.
root@poliva2-chef12-standalone-thor:~# chef-solo -v
Chef: 11.18.0.rc.1